README.md

```diff
@@ -20,6 +20,13 @@ END_METADATA -->
 [Vipps MobilePay Technical Documentation](https://developer.vippsmobilepay.com/docs/APIs/access-token-api/).
 <!-- END_COMMENT -->
 
+**Important:** Some of the new APIs, including the
+[Management API](https://developer.vippsmobilepay.com/docs/APIs/management-api/)
+and
+[Report API](https://developer.vippsmobilepay.com/docs/APIs/report-api/)
+use a new token endpoint. See
```

Comment:
Nitpicking: You don't need to use the new token endpoint, depending on your role. I think the current phrasing might lead to questions, to me it implies that everyone must use new endpoint to get access :thinking:

Management API: Management partners use new endpoint, partners & merchants use old endpoint.

Report API: Accounting partners use new endpoint, merchants use old, partners does not have access